distribution/docs/pom.xml

```diff
@@ -21,13 +21,16 @@
     </parent>
     <artifactId>docs</artifactId>
     <name>DDF DOCS</name>
+    <properties>
+        <skipTests>true</skipTests>
+    </properties>
     <build>
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
-                    <skip>true</skip>
+                    <skip>${skipTests}</skip>
```

Comment:
Seems like you only use the property here, maybe just get rid of the property and set this to `true`?